contracts/contracts/validator-registry/middleware/MevCommitMiddlewareStorage.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: BSL 1.1
+pragma solidity 0.8.26;
+
+import {IMevCommitMiddleware} from "../../interfaces/IMevCommitMiddleware.sol";
+import {EnumerableSet} from "../../utils/EnumerableSet.sol";
+import {IRegistry} from "symbiotic-core/interfaces/common/IRegistry.sol";
+
+abstract contract MevCommitMiddlewareStorage {
+
+    uint96 public constant SUBNETWORK_ID = 1;
+
+    uint64 public constant NETWORK_RESTAKE_DELEGATOR_TYPE = 0;
+    uint64 public constant FULL_RESTAKE_DELEGATOR_TYPE = 1;
+
+    uint64 public constant INSTANT_SLASHER_TYPE = 0;
+    uint64 public constant VETO_SLASHER_TYPE = 1;
+
+    IRegistry public networkRegistry;
+
+    IRegistry public operatorRegistry;
+
+    IRegistry public vaultFactory;
+
+    /// @notice The network address, which must have registered with the NETWORK_REGISTRY.
+    address public network;
+
+    /// @dev A period in seconds during which the mev-commit oracle can invoke slashing.
+    /// @notice This serves as the deregistration period for all of validator, operator, and vault records.
+    /// @notice This also serves as the number of seconds that a registered Vault's epochDuration must be greater than.
+    uint256 public slashPeriodSeconds;
+
+    address public slashOracle;
+
+    mapping(bytes blsPubkey => IMevCommitMiddleware.ValidatorRecord) public validatorRecords;
+
+    mapping(address operatorAddress => IMevCommitMiddleware.OperatorRecord) public operatorRecords;
+
+    mapping(address vaultAddress => IMevCommitMiddleware.VaultRecord) public vaultRecords;
+
+    mapping(address vault =>
+        mapping(address operator => EnumerableSet.BytesSet)) internal _vaultAndOperatorToValset;
+}
```

Comment:
Should there be a gap here for future storage extensions?

Reply:
Yes, nice catch!